client/src/pages/SignUp.tsx

```diff
@@ -1,28 +1,31 @@
-import { useCreateUserWithEmailAndPassword } from 'react-firebase-hooks/auth';
+import { useCreateUserWithEmailAndPassword, useUpdateProfile } from 'react-firebase-hooks/auth';
 import { auth } from '../firebase/connection';
 import { useNavigate } from 'react-router-dom';
-import classes from './Sign.module.scss';
 import Form, { FormOptions } from '../components/Form';
+import Header from '../components/Header';
 
 const SignUpPage = () => {
   const navigate = useNavigate();
-  const [createUserWithEmailAndPassword, user] = useCreateUserWithEmailAndPassword(auth);
+  const [createUserWithEmailAndPassword] = useCreateUserWithEmailAndPassword(auth);
+  const [updateProfile] = useUpdateProfile(auth);
 
-  const handleFormSubmit = ({ email, password }: FormOptions) => {
-    createUserWithEmailAndPassword(email as string, password as string);
+  const handleFormSubmit = async ({ name, email, password }: FormOptions) => {
```

Comment:
Kind of weird that we didn't mention error handling during authorization, or in any other process.
Maybe we should open an issue about that, I think it's kind of critical for the end user.

Reply:
גם זה משהו שאמור להיות מטופל על ידי המעצבת. אין את זה כרגע ואנחנו צריכים את הפורמים כך שהמשכתי הלאה